postgis/lwgeom_functions_analytic.c

```diff
@@ -223,6 +223,50 @@ Datum LWGEOM_line_interpolate_point(PG_FUNCTION_ARGS)
 
 	PG_RETURN_POINTER(result);
 }
+
+/***********************************************************************
+ * Interpolate a point along a line
+ * 3D version by vincent.mora@olandia.com
```

Comment:
My mistake, It's oslandia. It's corrected since commit 35b628f.

Where do you mean "in the header". This methods is not listed in the lwgeom_functions_analytic.h